understanding/22/consistent-help.html

```diff
@@ -43,49 +43,48 @@ <h2>Consistent Help</h2>
         <h2>Intent of Consistent Help</h2>
 
 
-        <p>The intent of this success criterion is to ensure users can find help for completing tasks on a Web site, when it is available. This is distinct from interface-level help, such as contextual help, features like spell checkers, and instructional text in a form. </p>
+        <p>The intent of this Success Criterion is to make it easier for users to find available help for a Web site. This is distinct from interface-level help, such as contextual help, spell checkers, and instructional text in a form. </p>
+
+        <p>At least one of the following ways to get help should be included on all websites:</p>
```

Comment:
i'd caution the use of the word "should" here, as it may give the impression that it tries to make a normative statement (while yes, the understanding doc is non-normative, readers may not understand the fine point)

Reply:
it may be that the "While it would be best..." sentence could be moved before the bullet list, and combined with the spirit of this to basically say (heavily paraphrased): normatively this SC doesn't specify the types of help, but as a best practice, these are the sorts of things that would be good to have...